clean-leather.lic

```diff
@@ -20,15 +20,11 @@ class CleanLeather
     ]
     args = parse_args(arg_definitions)
 
-    settings = get_settings
-    @hometown = get_settings.hometown
-    @bankbot_name = settings.bankbot_name
-    @bankbot_enabled = settings.bankbot_enabled
-    @bankbot_room_id = settings.bankbot_room_id
+    @settings = get_settings
```

Comment:
This does not need to be an instance var (`@`) since it is not used outside this function